mongodb/driver.go

```diff
@@ -117,6 +119,12 @@ func (m *MongoConnectionConfig) GetConnectionURI() (string, error) {
 		connectionString = fmt.Sprintf(connectionStringTemplateWithoutCreds, endpoint, m.Database)
 	}
 
+	if m.ReplicaSet != "" {
+		connectionString += fmt.Sprintf("?%s=%s", replicaSet, m.ReplicaSet)
+	} else {
+		connectionString += fmt.Sprintf("?%s=true", directConn)
```

Comment:
Do we really need constants for "replicaSet" and "directConnection"? Since they are only used here, I find it would be easier to read if we included them in the format string?

```suggestion
		connectionString += fmt.Sprintf("?replicaSet=%s", m.ReplicaSet)
	} else {
		connectionString += "?directConnection=true"
```

Reply:
No we don't need the constants per se

Reply:
changed